art-detail-pages/bear-zentangle-1.html

```diff
@@ -37,10 +37,31 @@
           </ul>
         </p>
         <div class="buy-now">
-          <a href="mailto:caitlin.daitch@gmail.com">Email for Custom Request</a>
+          <a href="#">Request Custom Art</a>
         </div>
       </div>
 
+      <div class="popup">
+        <a href="#"><img src="../images/close_icon.png" class="close-icon"></a>
+        <form id="custom-request" action="mailto:caitlin.daitch@gmail.com" enctype="text/plain">
+          <label>
+            <input type="text" placeholder="Name" />
+          </label><br>
```

Comment:
even though Luke demo'ed using the br's, try adding styles to the label to accomplish the layout you want instead -- for what you've got, it would mostly be as easy as making the label have `display: block`